test/memspace_numa.cpp

```diff
@@ -66,6 +82,10 @@ struct numa_memspace_provider_test : ::numa_memspace_test {
     void TearDown() override {
         ::numa_memspace_test::TearDown();
 
+        if (skip_this_test) {
```

Comment:
Skipping tests should be done before they begin (during SetUp()), not at the cleaning step (during TearDown()).

Reply:
So try and remove it ;-) I would not add it if it were not needed :-)